pbiviz.json

```diff
@@ -1,18 +1,18 @@
 {
   "visual": {
-    "name": "mapboxGLMap",
-    "displayName": "Mapbox Visual",
+    "name": "Intelematics - mapboxGLMap",
+    "displayName": "Intelematics - Mapbox Visual",
     "guid": "PBI_CV_EB3A4088_75C5_4746_9D8B_255A7B7ECD6D",
     "visualClassName": "MapboxMap",
-    "version": "20201201",
+    "version": "20200113-1",
```

Comment:
this we can change and should be used to track our internal version of this pbiviz

Reply:
The year is so 2020